### PR TITLE
dragonfly-reverb: 1.0.0 -> 1.1.4

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by JohnAZoidberg** [comment]
The update looks good, still builds.
But you only package the library - why not also copy the graphical application to the output? :)

File: pkgs/applications/audio/dragonfly-reverb/default.nix [hunk 1]
```diff
@@ -6,8 +6,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "michaelwillis";
     repo = "dragonfly-reverb";
-    rev = "1.0.0";
-    sha256 = "05m4hd8lg0a7iiia6cbiw5qmc4p8vbkxp2qh7ywaabawiwa9r24x";
+    rev = "1.1.4";
```
Comment (JohnAZoidberg) [inline pkgs/applications/audio/dragonfly-reverb/default.nix]: You can now use `pname = "dragonfly-reverb"` and `version = "1.1.4"`. Nix will automatically concatenate them.
Then you can do:
```suggestion
    rev = version;
```

and also `repo = pname`

**Comment by JohnAZoidberg** [comment]
The license was changed to GPL3: https://github.com/michaelwillis/dragonfly-reverb/commit/819696245495c9e6696d13c3fd16cf36342f68ef#diff-9879d6db96fd29134fc802214163b95a

**Comment by magnetophon** [comment]
Thanks for the feedback and good catch on the license!
All done.

**Pull request merged**